### ISSUE TITLE
Open TX 2.3.9 issue with sport telemetry update 

### ISSUE THREAD

**Issue by robertha64** [opentx/opentx#7767]
After flashing OpenTX version 2.3.9 to Radiomaster TX16S, all the sensors from Flight Controller (Omnibus F4V3 Inav) has random issues with updating (updating few times, then no updating about 20seconds and again updating and so on) and alarm "sensor lost" appears periodically. Tried flashing internal module with latest multimodule FW (1.3.1.9) - no ifluence - same behavior.

When flashed back opentx-tx16s-Radiomaster (7ee5b1d0) supplied with unit, the problem disappear and all sensor update as expected.

**Comment by 3djc** [comment]
I happen to work with the same radio and FC, no problem here whatsoever.  What radio protocol ?

**Comment by robertha64** [comment]
Frsky LBT 8ch

**Comment by 3djc** [comment]
Please reflash 2.3.9 from the sd card (not flashing with companion, put firmware in SD FIRMWARE folder, start radio in bootloader and flash), there is really no reason for this to happen.

**Comment by robertha64** [comment]
Flashed 2.3.9 from SD card, power cycle.
Same issue, few seconds updating all sensors, but then sensors from FC start loosing update and after about 1 minute I get the sensor lost alarm.

**Comment by robertha64** [comment]
Hm, weird after about 8 minutes its starts updating completely ?

**Comment by 3djc** [comment]
There is nothing I can think of on opentx side that would create that (not syaing it is not opentx, just that I cannot even find a place to start investigating). I'm guessing you ruled out proximity swamping and did the multi rf tune ?

**Comment by robertha64** [comment]
Yes, tuned to highest signal as described and TX is about 5 meters from RX. I have also Yaapu mapping Widget installed (and working when telemetry updates GPS).

**Comment by 3djc** [comment]
well you want to try to narrow down thngs a little. Please create a new  model with just default setup, bind, discover sensor and check what is happening

**Comment by robertha64** [comment]
OK I will do, just sending few pictures now from TX16 when it happened.

**Comment by robertha64** [comment]
Hm, not succeed to upload pic

**Comment by robertha64** [comment]
![IMG_0199](https://user-images.githubusercontent.com/39700425/85682236-61e1e800-b6cc-11ea-9d96-5c26d61a77be.JPG)
![IMG_0198](https://user-images.githubusercontent.com/39700425/85682276-6b6b5000-b6cc-11ea-9255-8d79421ec0f0.JPG)

**Comment by 3djc** [comment]
your rxbt is really low, is that intended ?

**Comment by robertha64** [comment]
No, can be FC issue as RX is powered from it?

**Comment by 3djc** [comment]
not sure, but this is at least an intrigingly low rxbt

**Comment by robertha64** [comment]
Find out - I did supply 5.2V on vbat connection of the FC, so internal regulator strugles to keep higher voltage on internal 5V rail. Now suppling 3S on vbat and rxbt is 5V. 

By the way the sensor update issue dissapear (before I change supply voltage), so I will probably close this as some magic temporary unknown glitch ?

**Comment by 3djc** [comment]
I have replicated your setup here and could not find any issue, so yes, propbably worth closing

**Comment by robertha64** [comment]
OK thank you for your help.

**Comment by diogo-f** [comment]
Today something similar to this happened with me. I'm using a Jumper T18 (with v2.3.8 officially adapted by jumper) and TBS Crossfire Micro TX V2 with firmware v3.72. 

I was flying and had a small crash, then I checked the TX screen and noticed the telemetry info on the TX was not updated with the info on my goggles. 
I rebooted the drone and when it connected it was still displaying the same old telemetry for a few seconds!! So this seems to be on the TX side, but if its happening randomly with other transmitters it seems to be on the radio and opentx... 

let me know if there is anything I can do/send to help, does opentx write logs on the sd card?

(I came to opentx to see if there is a new version with [t18 support and it will be in 2.3.10](https://www.open-tx.org/radios.html), I also have a new firmware for the Crossfire TX but some people are having issues with the new v4 so i'm waiting for that one to be stable.)
